Sources/BraintreePayPal/Public/BraintreePayPal/BTPayPalClient.h

```diff
@@ -16,35 +16,35 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  Domain for PayPal errors.
  */
-extern NSString *const BTPayPalDriverErrorDomain;
+extern NSString *const BTPayPalClientErrorDomain;
```

Comment:
Just a thought for all error naming - do we think the full client name prefix is valuable or would `BTPayPalErrorType` and `BTVenmoErrorType` be better?

Reply:
+1 would cut down on the verbosity some. Could be a separate ticket for simplifying the error type naming convention.

Reply:
I am 100% down for making this change in this PR, especially since it's already a breaking change. Will make this update.